xclim/indices/_conversion.py

```diff
@@ -538,3 +539,98 @@ def rain_approximation(
     prlp = pr - snowfall_approximation(pr, tas, thresh=thresh, method=method)
     prlp.attrs["units"] = pr.attrs["units"]
     return prlp
+
+
+@declare_units(
+    tas="[temperature]",
+    sfcWind="[speed]",
+    max_temp_thresh="[temperature]",
+    min_wind_thresh="[speed]",
+)
+def wind_chill_index(
+    tas: xr.DataArray,
+    sfcWind: xr.DataArray,
+    slow_wind_calc: bool = True,
+    mask_invalid: bool = True,
+    max_temp_thresh: str = "0 degC",
+    min_wind_thresh: str = "5 km/h",
+):
+    """Wind chill index.
+
+    The Wind Chill Index is an estimation of how cold the weather feels to the average person.
+    It is computed from the air temperature and the 10-m wind. As defined by the Environment and Climate Change Canada ([ECCC]_),
+    two equations exist, the conventionnal one and one for slow winds (usually < 5 km/h), see Notes.
+
+    Parameters
+    ----------
+    tas : xarray.DataArray
+      Surface air temperature.
+    sfcwind : xarray.DataArray
+      Suface wind speed (10 m).
+    slow_wind_calc : bool
+      If True (default), a "slow wind" equation is used where winds are slower than min_wind_thresh, see Notes.
+    mask_invalid : bool
+      Whether to mask values when the inputs are outside their validity range. or not.
+      If True (default), points where tas > max_temp_thresh are masked in the output.
+      If `slow_wind_calc` is also False, points where sfcWind < min_wind_thresh are masked.
+    max_temp_thresh : str
+      The maximal allowed temperature when computing the index. If mask_invalid is False,
+      points where tas is above this threshold are masked.
+    min_wind_thresh : str
+      The minimal wind speed for using the standard computation. If `slow_wind_calc` is True,
+      a slow-wind version of the index is used for points where sfcWind is below this threshold.
+      If it is False and `mask_invalid` is True, those points are simply masked.
+
```

Comment:
I feel this is a tad too complicated. It's not yet clear to me how to simplify this, but I think we should come up with something less flexible but simpler, ie method={"US", "CAN"}

Reply:
Ok. I was trying not to have this kind of switch, but it's true that 3 kwargs is a bit complicated.

I'd keep `mask_invalid`, but can replace the other 3 by `method`.